web/html/links/anchor-links.md

```diff
@@ -57,20 +55,20 @@ This is a
 
 [View CodePen](https://codepen.io/enkidevs/pen/NBpQzw)
 
+
 ---
 ## Practice
 
 Complete the following snippet in order for it to be valid:
 ```html
 Link to the
-<tag ???="???contact.html">
+<tag ???="#contact.html">
  contact section
 </tag> of the page.
 ```
 tag = ???
 
 * `href`
-* `#`
 * `a`
 * `style`
 * `p`
```

Comment:
How about we change the question to:

    ```html
    Link to the
    <??? ???="#contact.html">
     contact section
    <???> of the page.
    ```

    * `a`
    * `href`
    * `/a`
    * `style`
    * `p`

Reply:
@sourabhv having only `/a` as the closing tag in the answers makes it an obvious answer 🤷‍♂ 

Maybe something like?

    ```html
    Link to the
    <??? ???="#contact.html">
     contact section
    <???> of the page.
    ```

    * `a`
    * `href`
    * `/a`
    * `url`
    * `/p`

Reply:
How about we change the question to:

    ```html
    Link to the
    <??? ???="#contact.html">
     contact section
    <???> of the page.
    ```

    * `a`
    * `href`
    * `/a`
    * `style`
    * `link`
    * `/link`
    * `url`
    * `p`
    * `/p`

Reply:
That looks better, I've added it